build.gradle.kts

```diff
@@ -24,6 +25,13 @@ allprojects {
         maven {
             url = uri("https://repo.akka.io/maven")
         }
+        maven {
+            url = uri("https://maven.pkg.github.com/position-pal/shared-kernel")
+            credentials {
+                username = project.findProperty("gpr.user") as String? ?: EnvHelper.getEnv("GH_USERNAME", "dfd")
+                password = project.findProperty("gpr.key") as String? ?: EnvHelper.getEnv("GH_TOKEN", "fdfd")
```

Comment:
Maybe you forgot `dfd` and `fdfd` defaults?

Reply:
Yhea..that's akward, thanks for pointing it out